### ISSUE TITLE
Recommended changes resulting from automated audit

### ISSUE THREAD

**Issue by Jericho** [cake-contrib/Cake.Issues.MsBuild#27]
We performed an automated audit of your Cake addin and found that it does not follow all the best practices.

We encourage you to make the following modifications:

- [ ] The Cake.Core reference should be private. Specifically, your addin's `.csproj` should have a line similar to this: `<PackageReference Include="Cake.Core" Version="0.28.0" PrivateAssets="All" />`

Apologies if this is already being worked on, or if there are existing open issues, this issue was created based on what is currently published for this package on NuGet.org and in the project on github.

**Comment by pascalberger** [comment]
@Jericho Is it expected to recreate issues on every run, even if previous issue (#26) was closed and no new validations warnings were found?

**Comment by Jericho** [comment]
If the issue is still open, no new issue is created. 

I cleared my local cache and re-running the process to ensure that I correctly pick up latest changes and avoid the issue you pointed out earlier where I incorrectly flag your project as referencing an old Cake DLL when in fact you already upgraded to latest.

**Comment by pascalberger** [comment]
Mmh, this can be quite spammy as for example a repository might already have existing open issues for the validated points and therefore closes the audit issue.

**Comment by Jericho** [comment]
Agree, that's why this "create github issue" process is not scheduled to run. I run it manually. My think is to run it only once every time Cake is released with breaking changes.

**Comment by gep13** [comment]
@Jericho perhaps a note at the bottom of the automated issue that was created, i.e. 

> apologies if this is already being worked on, or if there are existing open issues, this issue was created based on what is currently published for this package on NuGet.org.

**Comment by Jericho** [comment]
Good suggestion. Adding it now...

**Comment by Jericho** [comment]
@pascalberger are you on Gitter right now. Probably easier to chat there...

**Comment by Jericho** [comment]
@pascalberger please do not close the issues. Give me some time to re-run the process to add the disclaimer at bottom and correct the false positive. Existing issues will be updated.

**Comment by gep13** [comment]
>@Jericho said...
Existing issues will be updated.

That is probably worth mentioning in the disclaimer as well.

**Comment by Jericho** [comment]
@gep13 Normally issues are not updated. I just updated them this one time to correct false positive (caused by me not clearing my cache and also to add disclaimer). #

**Comment by gep13** [comment]
Ah, gotcha, that makes sense.

**Comment by pascalberger** [comment]
Closing as this is not an issues since nupkg is created from nuspec file
